go-controller/pkg/ovn/ovn.go

```diff
@@ -125,6 +125,17 @@ func (oc *DefaultNetworkController) ensurePod(oldPod, pod *kapi.Pod, addPort boo
 		return nil
 	}
 
+	if config.HybridOverlay.Enabled {
+		node, err := oc.kube.GetNode(pod.Spec.NodeName)
```

Comment:
performance problem. This needs to use the watch factory/informer.

Is the check really necessary? We must already skip no host subnet nodes already somewhere. Maybe:

func (oc *DefaultNetworkController) addLogicalPort(pod *kapi.Pod) (err error) {
	// If a node does node have an assigned hostsubnet don't wait for the logical switch to appear
	switchName := pod.Spec.NodeName
	if oc.lsManager.IsNonHostSubnetSwitch(switchName) {
		return nil
	}

Reply:
`addLogicalPort` will only be invoked if the pod is a localzone pod. The code here is to skip handling the pods on a remote node. But I think I can use `IsNonHostSubnetSwitch` to determine if the node is an HO node.